.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 sudo: false
 node_js:
-  - 'node'
+  - 10
```

Comment:
I had to pin this at node v10 to alleviate issues in CI -- cause as yet is undetermined